FRIEND/FRIEND.cpp

```diff
@@ -211,7 +215,7 @@ class FRIEND : public PluginDelegate
 				if ( place == nullptr )
 					return 0;
 				
-				auto ea = place->toea();
+				auto ea = ((idaplace_t *)place)->ea;

```

Comment:
With IDA 6.8 SDK, it seems that ``toea()`` is not defined. This is the reason why I did this type casting.